Sync/Synchronizers/ClientsSynchronizer.swift

```diff
@@ -346,3 +345,98 @@ public class ClientsSynchronizer: TimestampedSingleCollectionSynchronizer, Synch
             >>> { deferMaybe(.Completed) }
     }
 }
+
+private extension UIDevice {
+    class func formFactorString() -> String {
+        var formfactor: String
+        
+        let phones: Set = [
```

Comment:
Nit: Super small thing but moving these Set definitions outside the extension and as a private file scoped variable would prevent the sets from being allocated every time we call `formFactorString()`. Minor perf improvement :P